### PR TITLE
Add schema.org vocabulary

### PR DESCRIPTION
**Pull request by ebremer**
Add schema.org vocabulary
remove capitalization from Web Access Control vocabulary

### CODE REVIEW

**Comment by ebremer** [comment]
added FOAF this morning

**Comment by afs** [comment]
Hi - a few process things:

* please could you changes to different vocabularies in different PRs especially changes like the capitalization alterations.
* The commits reflect your development - when in the codebase, log messages are more about the substantive changes (think about what someone wants to read in 2-5 years time!). If the intermediate could be squashed that would be great.

Jena already has FOAF.java (in org.apache.jena.sparql.vocabulary because i tis used in standard W3C test cases).
It was generated using schemagen from the FOAF schema document itself. schema also includes the items rdfs:comment as its javadoc.

**Comment by ebremer** [comment]
Will do.

File: jena-core/src/main/java/org/apache/jena/vocabulary/WAC.java [hunk 1]
```diff
@@ -17,30 +17,32 @@
...
+ *  <p>
  *	See <a href="https://www.w3.org/wiki/WebAccessControl">W3C WebAccessControl</a>.
  *  <p>
  *  <a href="http://www.w3.org/ns/auth/acl">Base URI and namepace</a>.
  */
-public class WAC {
     private static final Model m = ModelFactory.createDefaultModel();
     public static final String NS = "http://www.w3.org/ns/auth/acl#";
 
-    public static final Resource APPEND = m.createResource(NS+"Append");
```
Comment (afs) [inline jena-core/src/main/java/org/apache/jena/vocabulary/WAC.java:36]: I've extracted these changes and applied them directly.

**Pull request closed without merging**